src/tools/run-make-support/src/lib.rs

```diff
@@ -149,6 +149,46 @@ pub fn bin_name(name: &str) -> String {
     if is_windows() { format!("{name}.exe") } else { name.to_string() }
 }
 
+fn ar<P: AsRef<str>, P2: AsRef<str>>(obj_path: P, lib_path: P2, caller_line_number: u32) {
+    let mut ar = Command::new(env::var("AR").unwrap());
+    ar.current_dir(tmp_dir()).arg("crus").arg(lib_path.as_ref()).arg(obj_path.as_ref());
+    let output = ar.output().unwrap();
+    if !output.status.success() {
+        handle_failed_output(&ar, output, caller_line_number);
+    }
+}
+
+/// Builds a static lib (`.lib` on Windows MSVC and `.a` for the rest) with the given name.
+pub fn build_native_static_lib(lib_name: &str) {
```

Comment:
Discussion: we probably *should* return the path to the constructed static lib here, because subsequent code in `rmake.rs` would probably want to have the path to it?

Reply:
Sounds good to me. I don't see a reason to not do it so adding it. :)